inlang/source-code/sdk/src/generateProjectId.test.ts

```diff
@@ -0,0 +1,24 @@
+import { generateProjectId } from "./generateProjectId.js"
+import { describe, it, expect } from "vitest"
+import { openRepository } from "@lix-js/client"
+import { createNodeishMemoryFs } from "@lix-js/fs"
+
+describe("generateProjectId", async () => {
+	const repo = await openRepository("https://github.com/inlang/ci-test-repo", {
+		nodeishFs: createNodeishMemoryFs(),
+	})
```

Comment:
@janfjohannes just to double check: 

are we sure that we don't want to introduce `createVirtualRepo` with this change? See https://github.com/inlang/monorepo/issues/1898#issuecomment-1861852281, we can "ignore" github APIs. 

- the test is long running now because it makes a network request
- the test is fragile because it makes a network request

Reply:
```ts

const repo = await createVirtualRepo({
   // optional arguments
  origin: "https://mock.com"
})

Reply:
@samuelstroschein  replaced the network requests with mock data